include/manifold/manifold.h

```diff
@@ -270,7 +270,7 @@ class Manifold {
   size_t NumVert() const;
   size_t NumEdge() const;
   size_t NumTri() const;
-  size_t NumProp() const;
+  int NumProp() const;
```

Comment:
Why do you want to turn this into `int`?

Reply:
Because NumProp is of type int: https://github.com/elalish/manifold/blob/f1a86930164b360765c5e04d633aae4d617acdd1/src/impl.h#L37C5-L37C21

Reply:
I think the right thing is to update that to `size_t` rather than vice-versa.

Reply:
Okay, I thought about trying a simpler refactor, but if you say it is fine to change that to `size_t`, I'll do that later :smile:

Reply:
Certainly beats changing our public API 😄

Reply:
Yeah, we have things that are not yet `size_t` but probably should be. They are mostly internal, though.

Reply:
Sounds good, I think we can keep the internal things unchanged for now.